vaultlocker/shell.py

```diff
@@ -40,10 +40,17 @@ def _vault_client(config):
     """
     client = hvac.Client(
         url=config.get('vault', 'url'),
-        verify=config.get('vault', 'ca_bundle', fallback=True)
+        token=config.get('vault', 'token', fallback=None),
+        verify=config.get('vault', 'ca_bundle', fallback=True),
+        namespace=config.get('vault', 'namespace', fallback=None)
```

Comment:
No need to provide fallback in this case.

Reply:
Just double checked this, without the fallback, config.get will complain that the value is not defined, so another solution would be for me to put those in the defaults when initialising the ConfigParser object like here:
https://github.com/openstack-charmers/vaultlocker/blob/e4630e718354765fb523c1aa1be9b2dc9af8703e/vaultlocker/shell.py#diff-39a37307a7b5fc79ae5c062e4601bc04d633951537c31a1af57fe43f08878e25L207-R239
```diff
- config = configparser.ConfigParser()
+ config = configparser.ConfigParser({'kv_version': '1'})
```